tests/functional/workloads/cnv/test_vm_snapshot_cloning_ops.py

```diff
@@ -16,9 +17,40 @@ class TestVmSnapshotClone(E2ETest):
     """
 
     @workloads
-    @pytest.mark.polarion_id("OCS-6288")
+    @pytest.mark.parametrize(
+        argnames=["pvc_expand_before_clone", "pvc_expand_after_clone"],
+        argvalues=[
+            pytest.param(
+                False,
+                False,
+                marks=pytest.mark.polarion_id(
+                    "OCS-6288"
+                ),  # Polarion ID for no PVC expansion
+            ),
+            # pytest.param(
+            #     True,
+            #     False,
+            #     marks=pytest.mark.polarion_id(
+            #         "OCS-6326"
+            #     ),  # Polarion ID for expansion before clone
+            # ),
+            # pytest.param(
+            #     False,
+            #     True,
+            #     marks=pytest.mark.polarion_id(
+            #         "OCS-6326"
+            #     ),  # Polarion ID for expansion after clone
+            # ),
+        ],
+    )
     def test_vm_clone(
```

Comment:
as discussed, please rename the test name according to the test steps, you are covering both clone and expansion

Reply:
Done